These fields have been unused for many years, but it is only now,
that some compilers (clang-3.2) complain about it...

	-mi

--- fpx/fpximgvw.h	2007-11-02 03:10:05.000000000 -0400
+++ fpx/fpximgvw.h	2012-10-12 12:28:39.000000000 -0400
@@ -300,6 +300,4 @@
 
         PFileFlashPixView*  filePtr;      // The image file descriptor
-        OLEStorage*   sourceStorage;    // Source FPX Image storage
-        OLEStorage*   resultStorage;    // Result FPX Image storage
 
         // Geometric parameters
